packages/api-clients/open-api/m2mGatewayApi.yml

```diff
@@ -3364,6 +3364,22 @@ paths:
         - tenants
       operationId: getCertifiedAttributes
       description: Retrieve certified attributes for a Tenant
+      parameters:
+        - in: query
+          name: offset
+          required: true
+          schema:
+            type: integer
+            format: int32
+            minimum: 0
+        - in: query
+          name: limit
+          required: true
+          schema:
+            type: integer
+            format: int32
+            minimum: 1
+            maximum: 50
       responses:
```

Comment:
Here we should also add errors that can pass through from the process, in this case, let's add a `404` because the `getTenant` call can return a tenantNotFound mapped to 404